src/lib/error-handling.ts

```diff
@@ -0,0 +1,9 @@
+const throwError = (error: unknown, message?:string): void => {
```

Comment:
This function doesn't really do anything. You can just use `throw new Error(message)` in all the places you're calling this function.